admin/bower.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "OSF_admin",
+  "dependencies": {
+    "jquery": "^1.11.2",
```

Comment:
Actually, you shouldn't need bower at all for this project. Bower is [no longer maintained](https://github.com/bower/bower/pull/1748#issuecomment-150104311) and is redundant with npm.

Just use `package.json`.